### DISCUSSION TITLE
为什么使用不了

### DISCUSSION THREAD

**Discussion by QC514** [BurnHair/docs#135] [分享讨论]
![image](https://github.com/BurnHair/docs/assets/119123314/1d9aee05-4727-4b72-801e-634c93a90a7f)

**Comment by BennyThink** [comment]
https://github.com/BurnHair/docs/blob/master/lobe-chat%20%E5%92%8C%20ChatGPT-Next-Web%20%E4%BD%BF%E7%94%A8.md#%E9%85%8D%E7%BD%AE-1

参考这里进行配置

**Reply by QC514** [reply]
好的，解决了
